src/test/resources/application.properties

```diff
@@ -0,0 +1,54 @@
+info.app.name=${spring.application.name}
```

Comment:
Are these test values different from defaults in YAML file? Is that really needed? Maybe default would do for testing purpose.

Reply:
currently no yaml, do you mean our one from fpl-ccd-configuration